packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -747,6 +749,10 @@ export interface CallCompositeStrings {
     startCaptionsButtonOnLabel?: string;
     startCaptionsButtonTooltipOffContent?: string;
     startCaptionsButtonTooltipOnContent?: string;
+    TagsSurveyCancelButtonAriaLabel: string;
```

Comment:
these are PascalCase where the rest of the props are camelCase. lowercase the first letter in each of these